data/data-pipeline/data_pipeline/score/score_m.py

```diff
@@ -56,6 +56,22 @@ def _combine_island_areas_with_states_and_set_thresholds(
             [column_from_island_areas, column_from_decennial_census]
         ].mean(axis=1, skipna=True)
 
+        # Create a percentile field for use in the Islands / PR visualization
+        # TODO: move this code
+        # In the code below, percentiles are constructed based on the combined column
+        # of census and island data, but only reported for the island areas (where there
+        # is no other comprehensive percentile information)
+        return_series_name = (
```

Comment:
So a true nit comment -- I think if we have to do this, it makes sense to collapse this part of the code:

```
        # Calculate the percentile threshold raw value.
        raw_threshold = np.nanquantile(
            a=df[combined_column_name], q=threshold_cutoff_for_island_areas
        )

        logger.info(
            f"For combined field `{combined_column_name}`, "
            f"the {threshold_cutoff_for_island_areas*100:.0f} percentile cutoff is a "
            f"raw value of {raw_threshold:.3f}."
        )

        threshold_column_name = (
            f"{column_from_island_areas} exceeds "
            f"{threshold_cutoff_for_island_areas*100:.0f}th percentile"
        )

        df[threshold_column_name] = (
            df[column_from_island_areas] >= raw_threshold
        )
```

From here, I *think* we could simply do: 
```
        threshold_column_name = (
            f"{column_from_island_areas} exceeds "
            f"{threshold_cutoff_for_island_areas*100:.0f}th percentile"
        )

        df[threshold_column_name] = (
            df[return_series_name] >= threshold_cutoff_for_island_areas
        )
```

And we would not have to:
(1) calculate the threshold directly
(2) use the threshold to calculate which fields get included
(3) only treat the island areas -- the default here for NaN is `false` I think (`NaN` equals nothing, I'm pretty sure)

I'm not sure this actually matters,  though, especially given the refactor on the horizon. But, just a thought :)

Reply:
Switching to just using the 2nd code block in your comment makes a lot of sense me. I assume this code would come after you set `return_series_name`, right? 

Btw, this is not the best process, but in our absence of tests (sigh) you can compare the before and after score files using the notebook `data/data-pipeline/data_pipeline/ipython/compare_two_score_files_for_differences.ipynb`. Just give it two paths to different score files and it'll give you a pretty good readout on whether there are any differences.

Reply:
If you don't want to re-run `main` to get an old score file, you can get a `usa.csv` output from `"https://justice40-data.s3.amazonaws.com/data-pipeline"`, which is defined in `data/data-pipeline/settings.toml` and follows the same directory structure as your score outputs locally.

Reply:
ty ty! will do